vllm/assets/dashboards/overview.json

```diff
@@ -750,21 +944,21 @@
                 "height": 3,
                 "width": 12,
                 "x": 0,
-                "y": 11
+                "y": 15
             }
         },
         {
             "definition": {
-                "background_color": "vivid_blue",
+                "background_color": "vivid_orange",
                 "layout_type": "ordered",
                 "show_title": true,
                 "title": "CPU Usage",
                 "type": "group",
                 "widgets": [
                     {
                         "definition": {
-                            "background_color": "blue",
-                            "content": "Dive deeper to see how vLLM is using the CPU.",
+                            "background_color": "orange",
+                            "content": "You can save on some infrastructure costs by doing some of the work on CPUs instead of GPUs. In this section you can see how much of your vLLM load ends up sent to CPU for processing.",
```

Comment:
```suggestion
                            "content": "You can save on some infrastructure costs by doing some of the work on CPUs instead of GPUs. In this section, you can see how much of your vLLM load ends up being sent to the CPU for processing.",
```